src/main/service/delete-definition-service.ts

```diff
@@ -0,0 +1,35 @@
+import * as request from "superagent";
+import * as config from "config";
+import { Logger } from "@hmcts/nodejs-logging";
+
+export function deleteDefinition(req) {
+  const logger = Logger.getLogger(__filename);
+  // Note: Not a typo - the base URLs for creating and deleting a Definition are exactly the same, hence reusing the
+  // create one
+  const url = config.get("adminWeb.createdefinition_url");
```

Comment:
Thinking maybe we shall introduce another url in config.  That we you can save your L7..L8

Reply:
OK, I'll separate into its own config variable